### ISSUE TITLE
Restrict API key values

### ISSUE THREAD

**Issue by psiinon** [zaproxy/zaproxy#2663]
At the moment it is possible to self XSS yourself by using a suitable attack in the API key.
As ZAP is currently a single user application this isnt really a bit issue, but when ZAP supports multiple users then it could be a problem.

**Comment by psiinon** [comment]
Note that this will no longer cause an XSS due to the CSP applied. Do we still need to restrict the values?

**Comment by thc202** [comment]
I don't think we need to restrict the values, although we must make sure that we are properly escaping/encoding it where needed.
